<ui version="4.0" >
 <class>ShellHistoryDialog</class>
 <widget class="QDialog" name="ShellHistoryDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Shell History</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QListWidget" name="historyList" >
     <property name="font" >
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QPushButton" name="deleteButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Delete the selected entries</string>
       </property>
       <property name="text" >
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Copy the selected entries to the current editor</string>
       </property>
       <property name="text" >
        <string>C&amp;opy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="executeButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Execute the selected entries</string>
       </property>
       <property name="text" >
        <string>&amp;Execute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reloadButton" >
       <property name="toolTip" >
        <string>Reload the history</string>
       </property>
       <property name="text" >
        <string>&amp;Reload</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>72</width>
         <height>208</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>historyList</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>copyButton</tabstop>
  <tabstop>executeButton</tabstop>
  <tabstop>reloadButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShellHistoryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>333</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel" >
     <x>323</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShellHistoryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>167</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel" >
     <x>169</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
